docs/cluster-scope/add_resource_quotas.md

```diff
@@ -0,0 +1,40 @@
+# Adding Resource Quotas
+
+To add a resource quota follow the steps outlined below:
+
+# Adding a resource quota tier to a namespace
+
+Select a tier from [this list][quotas].
+
+Now add this quota to the namespace. For example, if you want to add `small` tier quota to the namespace `argocd` run:
+
+```bash
+# From the root of this repo
+$ cd cluster-scope/base/namespaces/argocd
+$ kustomize edit add component ../../../components/resourcequotas/small
+```
+
+# Adding a custom resource quota to a namespace
+
+Create your [own resource][quotas_doc]. Then add it to your namespace, for example if adding `${QUOTA_LOCATION}` to namespace `argocd` you would run:
+
+```bash
+# From the root of this repo
+$ cd cluster-scope/base/namespaces/argocd
+$ kustomize edit add resource ${QUOTA_LOCATION}
```

Comment:
Let's be more specific here. Let's require users to name their quota file `resourcequota.yaml` and use `.metadata.name = custom`. That way we know, even when viewed from console, what the quota "type" it is. It will also keep the repo tidy.

WDYT?

Reply:
Good point -- this actually poses a question I didn't think of, where should these custom quotas be stored? Should they be in `cluster-scope/components/resourcequotas/` ? If they are named `resourcequota.yaml` for every one, then they would have to be in some path that distinguishes them by namespace, or team name, etc. like: 

`cluster-scope/components/resourcequotas/custom/<namespace>/resourcequota.yaml`

Reply:
I actually think we should keep these custom quotas together with the namespace. Unless we want a separate `kustomization.yaml` for each - because as outlined in this PR, we basically want them to be imported into the `namespaces/ABC/kustomization.yaml`. I don't think it should look like:

```yaml
---
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

namespace: argocd

resources:
  - namespace.yaml
  - ../../resourcequotas/custom/argocd/resourceqouta.yaml

components:
  - ../../../components/project-admin-rolebindings/operate-first
  - ../../../components/monitoring-rbac

```

That doesn't look right to me.

1. Either keep them within the `namespaces/ABC` folder
2. Or do it like we do with `operatorgroups` and have a distinct folder and a `kustomization.yaml` and everything and import it to an overlay separately.

I think I kinda prefer the first option, because I don't want to risk not importing the quota that is a separate to the namespace and then not enforcing it. On the other hand that's not consistent with the repo layout...

Reply:
> On the other hand that's not consistent with the repo layout... 
This is my issue, consistency is nice for when we add automation. But I think making an exception in this case for namespaces is fine. Let's go with the first option and see where it takes us, we can always change it up.

Reply:
updated